src/database/sql_database.cc

```diff
@@ -275,7 +275,7 @@ std::vector<std::shared_ptr<SQLDatabase::AddUpdateTable>> SQLDatabase::_addUpdat
     }
 
     if (obj->isContainer() && op == Operation::Update && obj->isVirtual()) {
-        std::string dbLocation = addLocationPrefix(LOC_VIRT_PREFIX, obj->getLocation());
+        fs::path dbLocation = addLocationPrefix(LOC_VIRT_PREFIX, obj->getLocation());
         cdsObjectSql["location"] = quote(dbLocation);
```

Comment:
The result of `addLocationPrefix` should be handled as string because it's no filesystem path anymore. Maybe change inside.

Reply:
Sure. But addLocationPrefix returns fs::path.

Reply:
Then your code is fine.